### DISCUSSION TITLE
此项目长久没有维护，如何能够继续维护此项目？

### DISCUSSION THREAD

**Discussion by cmys1109** [Excaive/miraicle#26] [General]
## 此仓库已经长久没有任何更新动作
自从2021年10月开始，此项目仓库和miraicle的作者GitHub一直都没有任何的更新。虽然当下miraicle仍然能够正常使用，但是仍然有许多可以改进的地方。可是现存提交的两个PR长期以来都没有被受理，讨论区也长久没有新帖出现，个人感觉此项目已经被遗弃。

## 希望miraicle能够继续更新
我选择使用miraicle1的理由主要是：
1. 选择使用python相较于其他语言更加方便
2. 文档较为友好
3. API设计的易用、易懂

但是miraicle的功能并不是很完善，比如说缺少了处理好友申请、加入群聊申请、被邀请入群申请的接收和处理，由于我需要使用这方面的功能，我便自己编写了这方面的API，但是发现本项目一直都没有反应，甚至是作者的GitHub账号都没有贡献的，不知道作者是否不再使用此GitHub账号了？如果我提交pr，是否也会一直没有被处理？

我想如果可以，作者可以处理一下PR。如果实在没用精力维护是否可以考虑转让仓库。

我也没有找到作者的任何联系方式，无法直接与作者沟通，如果有知情人士可以在社区里交流。

**Comment by LiteCat0905** [comment]
你好我虽然不知道这个项目的主人联系方式，从主页看这个作者2022年没有任何一次提交。别说放弃这个项目了也许可能已经放弃这个帐号了吧（sosad

**Reply by cmys1109** [reply]
哎，我就是想要是作者放弃了这个项目还好还能转手，但是这GitHub账号不用了、又没有联系方式，想接手也没办法。
